docs/core/additional-tools/wcf-web-service-reference-guide.md

```diff
@@ -73,9 +73,15 @@ When these processes complete, you can create an instance of the generated WCF c
 
 ## Next steps
 
+### See also
+
+- [Get started with Windows Communication Foundation applications](https://docs.microsoft.com/en-us/dotnet/framework/wcf/getting-started-tutorial)
```

Comment:
```suggestion
- [Get started with Windows Communication Foundation applications](../../../framework/wcf/getting-started-tutorial.md)
```

Reply:
@gewarren I took your suggestion, but on the markdown diff the relative links don't seem to work as expected. Is there any way to verify this is pointing to the right page? If the current document is in "docs/core/additional-tools" and the document we're linking is in "docs/framework/wcf shouldn't the link be ../../framework/wcf"?

Reply:
@dasetser, I think you're correct. It should `../../` not `../../../`. You can push a commit and see if it will redirect you correctly on GitHub diff.

Reply:
@Youssef1313 That does look to have fixed that link. Thanks!